conf/hdfs-site.xml

```diff
@@ -24,72 +24,72 @@
 
   <property>
```

Comment:
Please don't modify the original conf/hdfs-site.xml file.This will create a bug for users that are not using Mesos DNS. We should leave the original files intact and only add examples for users who want to use Mesos DNS.